cuqi/experimental/mcmc/_conjugate.py

```diff
@@ -9,19 +9,21 @@
 class ConjugateNew(SamplerNew):
     """ Conjugate sampler
 
-    Sampler for sampling a posterior distribution where the likelihood and prior are conjugate.
+    Sampler for sampling a posterior distribution where the likelihood and prior are conjugate to each other - denoted as a conjugate pair.   
```

Comment:
```suggestion
    Sampler for sampling a posterior distribution which is a so-called "conjugate" distribution, i.e., where the likelihood and prior are conjugate to each other - denoted as a conjugate pair.   
```

Reply:
I like the definition given later of "conjugate distribution". Wouldn't it make sense to use this term here as well?

Reply:
Yes. Sounds goodd